src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1458,7 +1471,9 @@ export class HtmlVideoPlayer {
     /**
          * @private
          */
-    renderTracksEvents(videoElement, track, item, targetTextTrackIndex = PRIMARY_TEXT_TRACK_INDEX) {
+    async renderTracksEvents(videoElement, track, item, targetTextTrackIndex = PRIMARY_TEXT_TRACK_INDEX) {
+        const { userSettings } = await import('../../scripts/settings/userSettings');
```

Comment:
Sorry, my bad: we need to specify an object.
```suggestion
        const { currentSettings: userSettings } = await import('../../scripts/settings/userSettings');
```

We could also use `const userSettings = await import('../../scripts/settings/userSettings');`, but I think we should move away from [those fallbacks](https://github.com/jellyfin/jellyfin-web/blob/12d2cf31e2442bcadfa502cf98a8d115876b67dc/src/scripts/settings/userSettings.js#L665-L711).